bin/fetch-configlet

```diff
@@ -10,9 +11,13 @@ from urllib.request import urlopen
 from urllib.error import HTTPError, URLError
 
 
+logging.basicConfig(level=logging.INFO)
+logger = logging.getLogger(__name__)
```

Comment:
On-record, I usually avoid using `__name__` for logger as a style preference, as the logger name appears in the default log message format.

Reply:
That may not be the best habit ... `__file__` isn't consistent across operating systems, and isn't even always present, so you may just bust your file irrevocably with a confusing NameError, but you also may end up with a miles-long absolute path in every logging message. That's on top of breaking the logging hierarchy functionality itself, which could be a issue for downstream consumers.

Reply:
I don't typically use `__file__` either; I think here I used it because I knew the primary use case would be from the parent directory and no further. Normally, I use constant names (i.e. `getLogger("fetch-configlet")`). I probably should have just gone that route this time as well.